### PR TITLE
Updated changelog

### PR DESCRIPTION
**Pull request by scofalik**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Updated changelog for v44.1.0.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,23 +11,23 @@ We are pleased to announce the latest CKEditor 5 release, focusing on performanc
...
     * **Inline mode improvements**: We addressed a problem where comment annotations in inline mode did not close properly when clicking elsewhere in the content.
     * **Thread management**: We resolved an issue where creating a new thread was not interrupted when the corresponding marker was removed from the content, ensuring better stability during collaborative editing.
 * **Revision History update**:
-    * **Restore functionality**: We disabled the ability to restore the current (edited and not saved) revision, as it was illogical and led to some non-obvious behaviors.
+    * **Restore functionality**: We disabled the ability to restore the current (edited, not saved) revision, as it represents current content, so there is nothing to restore. At the same time, using it led to some non-obvious behaviors.
 * **Image handling**: We resolved an issue where images in the uploading state could be deleted when dragged and dropped within the editor. Keep dragging, even when it is not there 🙈.
 
-### 🎄
+### 🎄 **Happy holidays!**
```
Comment (scofalik) [inline CHANGELOG.md:30]: This looked like some random out of place emoji.

**Review by godai78 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (godai78) [reply CHANGELOG.md:30]: How about we stop using emojis in technical docs at all?

Comment (pomek) [reply CHANGELOG.md:30]: As we need this changelog, I'd like to discuss your proposal Bartek.

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**